pkg/deploy/apis/apps/v1/conversion.go

```diff
@@ -119,10 +118,5 @@ func addConversionFuncs(scheme *runtime.Scheme) error {
 		return err
 	}
 
-	if err := scheme.AddFieldLabelConversionFunc("v1", "DeploymentConfig",
-		oapi.GetFieldLabelConversionFunc(newer.DeploymentConfigToSelectableFields(&newer.DeploymentConfig{}), nil),
-	); err != nil {
-		return err
-	}
```

Comment:
This doesn't have a special case to map "name" to "metadata.name" for field selection.